corehq/motech/openmrs/repeater_helpers.py

```diff
@@ -363,7 +364,7 @@ def get_form_question_values(form_json):
     """
     Returns question-value pairs to result where questions are given as "/data/foo/bar"
 
-    >>> get_form_question_values({'form': {'foo': {'bar': 'baz'}}})
+    >>> get_form_question_values({'form': {b'foo': {b'bar': b'baz'}}})
```

Comment:
Do we have the constraint that form questions be ascii only?  I guess this would've failed already if there were any non-bytestring-compatible questions.  I'm wondering if the tests should've been updated to use unicode rather than these.

Reply:
Form Builder doesn't allow non-ASCII characters, e.g. "`avons-nous_un_problème is not a legal Question ID. It must start with a letter and contain only letters, numbers, and '-' or '_' characters.`" ... (The XForm standard does allow questions to have non-ASCII characters in their names -- that example would be fine, because "è" is a letter.)